unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/KernelCommunication/WebSocketCommunication/DCLWebSocketService.cs

```diff
@@ -32,6 +36,23 @@ private void SendMessageToWeb(string type, string message)
 #endif
     }
 
+    private void SendBinaryMessageToWeb(string sceneId, byte[] data)
+    {
+#if (UNITY_EDITOR || UNITY_STANDALONE)
```

Comment:
Why is this only relevant to editor/standalone?

Reply:
that is for sending messages through websocket, which we are only using for editor and desktop version.
and the webgl message goes through `JSFunctions.jslib`